src/containers/add-edit-container.js

```diff
@@ -2,55 +2,55 @@
 
 import { connect } from 'react-redux';
 import {
-    setSidebarMode,
-    setMarkdownGuideVisibility,
-    toggleSidebarCollapsed,
-    setPageTitlePrefix,
-    eventSavedSuccessfully,
-    eventSaveFailed,
-    eventDeletedSuccessfully,
-    eventDeleteFailed,
+  setSidebarMode,
+  toggleSidebarCollapsed,
+  setPageTitlePrefix,
+  getEventDataViaUrlParams,
+  deleteCurrentEvent,
+  eventSavedSuccessfully,
+  eventSaveFailed,
 } from '../data/actions';
 import AddEditEventPage from '../pages/add-edit/add-edit-page';
 
 // This function passes values/objects from the Redux state to the React component as props
 const mapStateToProps = state => {
-    return {
-        general: state.general,
-        // events: getVisibleEvents(state.events, state.labels),
-        possibleLabels: state.labels.labelList,
-        markdownGuideVisible: state.addEdit.markdownGuideVisible
-    }
+  return {
+    general: state.general,
+    eventData: state.events.current,
+    possibleLabels: state.labels.labelList,
+  }
```

Comment:
Are we going with [Airbnb semi style](https://github.com/airbnb/javascript#semicolons)? Currently the sources are inconsistent…

Reply:
Yeah. I figured I'd do a massive reformatting commit in the near future. Trying to minimize line changes right now, so there's some inconsistency.